### PR TITLE
assignmentdone by hira

### PR DESCRIPTION
**Pull request by hira-sajid**
I have completed

### CODE REVIEW

**Review by CodingSpecies — CHANGES_REQUESTED** [review]
@hira-sajid please can you explain the changes you have made please. Right now, all I see is that the free code camp link has been copied and pasted here: 
image

If you need any help feel free to ask below. Thank you.

**Comment by AvidCoder101** [comment]
@hira-sajid do you need any help :)

File: resources/LearnPython.md [hunk 1]
```diff
@@ -12,4 +12,7 @@
...
 | [Kaggle](https://www.kaggle.com/learn/python)                                                                                                                | Kaggle provides a brief overview of Python syntax, variable assignment, and arithmetic operators.                                                                                                                                                                                        |
+| [freeCodeCamp](https://www.freecodecamp.org/learn/scientific-computing-with-python/)                                                                         | freeCodeCamp covers Python fundamentals like variables, loops, conditionals, and functions. Then you'll quickly ramp up to complex data structures, networking, relational databases, and data visualization.                                                                                                                                
+
+                                                                                                                                                                                        |
```
Comment (cadamini) [inline resources/LearnPython.md:17]:
```suggestion
| [freeCodeCamp](https://www.freecodecamp.org/learn/scientific-computing-with-python/) | freeCodeCamp covers Python fundamentals like variables, loops, conditionals, and functions. Then you'll quickly ramp up to complex data structures, networking, relational databases, and data visualization.  |
```

There have been extra tabs.

Comment (cadamini) [reply resources/LearnPython.md:17]: It's also on line 18 (master?)

**Pull request closed without merging**